internal/services/nginx/nginx_deployment_resource.go

```diff
@@ -260,6 +253,17 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 		"tags": commonschema.Tags(),
 	}
 
+	if !features.FivePointOhBeta() {
+		resource["managed_resource_group"] = &pluginsdk.Schema{
+			Deprecated:   "The `managed_resource_group` field has been deprecated and will be removed in v5.0 of the AzureRM Provider.",
```

Comment:
I think it would be good to specify that this property isn't supported in the API anymore, so users are aware that they can't expect the same behaviour as before despite the deprecation
```suggestion
			Deprecated:   "The `managed_resource_group` field isn't supported by the API anymore and has been deprecated and will be removed in v5.0 of the AzureRM Provider.",
```

Reply:
👍